addons/website/static/src/js/editor/wysiwyg_multizone.js

```diff
@@ -256,6 +251,27 @@ var WysiwygMultizone = Wysiwyg.extend({
             }.bind(this));
         }.bind(this));
     },
+    /**
+     * Setup editable attributes on the given element.
+     *
+     * This is also used when element is updated/replaced
+     * after the editor is initialized.
+     *
+     * @public
+     * @param {jQueryElement} $target
+     */
+    setupEditable: function ($target) {
+        var $editable = $target || this.$el;
+        if ($target) {
+            $target.find(this.selectorEditableArea).attr('contenteditable', true);
+        } else {
+            this._getEditableArea().attr('contenteditable', true);
```

Comment:
Feels like the the `_getEditableArea` method should simply be able to receive a `$target` too instead of using `this.selectorEditableArea` by hand here

Reply:
In our case `$target` is not in the DOM yet, so `_getEditableArea` will not will not return new grid. 
If i replace the product grid before `_setupEditable` , grid will not be editable because of this https://github.com/odoo/odoo/blob/master/addons/website/static/src/js/editor/editor_menu.js#L139

Reply:
I was saying to give `$target` as argument to `_getEditableArea` so that the `this.selectorEditableArea` code is not duplicated.

Reply:
My bad. Updated the code according to your suggestion.